html-gpgs/src/de/golfgl/gdxgamesvcs/GpgsClient.java

```diff
@@ -267,17 +267,15 @@ private native void refreshDisplayname() /*-{
 
     protected void onDisplayName(String displayName) {
         this.displayName = displayName;
-        if (gsListener != null)
```

Comment:
Why is this OnSessionActive callback here? Seems weird when we receive a new displayname we trigger a session active event? Is this something that is needed in HTML GPGS stuff? I decided to remove this because I thought it is illogical for this to be here.

Reply:
Before your change, player display name was guarantueed to be available when game service was connected. Thus, the connection state was set to connected after the display name was received.

Reply:
Okay, I will revert that change. Thanks for the clarification.